Xamarin.PropertyEditing.Windows/DateToTextConverter.cs

```diff
@@ -0,0 +1,32 @@
+using System;
+using System.Globalization;
+using System.Windows;
+using System.Windows.Data;
+using System.Windows.Markup;
+using Xamarin.PropertyEditing.Common;
+
+namespace Xamarin.PropertyEditing.Windows
+{
+	[ValueConversion (typeof (Date), typeof (string))]
+	internal class DateToTextConverter : MarkupExtension, IValueConverter
+	{ 
+		public object Convert (object value, Type targetType, object parameter, CultureInfo culture)
+			=> !(value is Date dateValue) ? DependencyProperty.UnsetValue

```

Comment:
The formatting is a little funny to me.  Maybe just drop the `DependencyProperty.UnsetValue` down a line?